doc/reference/core-builtin.md

```diff
@@ -1764,202 +1545,186 @@ that extends the Gambit builtin exception record type.
+Returns true if the object *obj* is an exception object raised by the builtin `error`
 procedure. Note that this is separate from instances of `error::t`; nomenclature.
 
 ### error-message
-::: tip usage
-```
-(error-message obj)
-=> string
+``` scheme
+(error-message obj) -> string
+
+  obj := any object
 ```
-:::
 
-Returns the message associated with an error instance. If the object is not
+Returns the message associated with an error instance. If the object *obj* is not
 an error instance, it returns the result of `(display-exception obj)`.
 
 ### error-irritants
-::: tip usage
-```
-(error-irritants obj)
-=> list or #f
+``` scheme
+(error-irritants obj) -> list | #f
+
+  obj := any object
 ```
-:::
 
-Returns the irritants associated with an error instance. If the object is
+Returns the irritants associated with an error instance. If the object *obj* is
 not an error instance, it returns `#f`.
 
 ### error-trace
-::: tip usage
-```
-(error-trace obj)
-=> any
+``` scheme
+(error-trace obj) -> any
+
+  obj := any object
 ```
-:::
 
-Returns the trace associated with an error instance. If the object is not
+Returns the trace associated with an error instance. If the object *obj* is not
 an error instance, it returns `#f`.
 
 ## Miscellaneous Procedures
 
 ### values-count
-::: tip usage
-```
-(values-count obj)
-=> fixnum
+``` scheme
+(values-count obj) -> fixnum
+
+  obj := any object
 ```
-:::
 
-Counts the values in the object. If the object is not multiple `values`,
+Counts the values in the object *obj*. If the object is not multiple `values`,
 it is considered a single value.
 
 ### values-ref
-::: tip usage
-```
-(values-ref obj n)
+``` scheme
+(values-ref obj n) -> any
+
   n := fixnum
-=> any
 ```
-:::
 
-Returns the `n`th value in the object. If the object is not multiple `values`,
+Returns the *n*th value in the object *obj*. If the object is not multiple `values`,
 it returns the object itself.
 
 ### values-&gt;list
-::: tip usage
-```
-(values->list obj)
-=> list
+``` scheme
+(values->list obj) -> list
+
+  obj := any object
 ```
-:::
 
-Converts multiple `values` to a list. If the object is not multiple `values`,
+Converts multiple `values` to a list. If the object *obj* is not multiple `values`,
 it returns a list containing the object.
 
 ### subvector-&gt;list
-::: tip usage
-```
-(subvector->list obj [start = 0])
-=> list
+``` scheme
+(subvector->list obj [start = 0]) -> list
+
+  obj   := any object
+  start := number
 ```
-:::
 
-Coverts a vector-like object to a list, starting from field `start`.
+Coverts a vector-like object *obj* to a list, starting from field *start*.
 
 ### vector-map
-::: tip usage
-```
-(vector-map f . vectors)
-  f := procedure
+``` scheme
+(vector-map f . vectors) -> any
+
+  f       := procedure
   vectors := list of vectors
-=> any
 ```
-:::
 
-Map for vectors.
+Map for vectors. Traverses vector *vectors* and applies procedure *f* to each
```

Comment:
Not just a vector, a list of vectors, f takes multiple params with multiple vectors specified, stopping after the shortest vectors runs out of elems.